The Straits Times.js

```diff
@@ -0,0 +1,286 @@
+		'Lim Min Zhang': { first: 'Min Zhang', last: 'Lim' },
+		'Lim Rei Enn': { first: 'Rei Enn', last: 'Lim' },
+		'Lim Ruey Yan': { first: 'Ruey Yan', last: 'Lim' },
+		'Lim Yaohui': { first: 'Yaohui', last: 'Lim' },
+		'Lin Yangchen': { first: 'Yanchen', last: 'Lin' },
+		'Loh Guo Pei': { first: 'Guo Pei', last: 'Loh' },
+		'Low Lin Fhoong': { first: 'Lin Fhoong', last: 'Low' },
+		'Mok Qiu Lin': { first: 'Qiu Lin', last: 'Mok' },
+		'Ng Huiwen': { first: 'Huiwen', last: 'Ng' },
+		'Ong Sor Fern': { first: 'Sor Fern', last: 'Ong' },
+		'Quah Ting Wen': { first: 'Ting Wen', last: 'Quah' },
+		'Raynold Toh YK': { first: 'Toh', last: 'Raynold, YK' },
+		'Rebecca Tan Hui Qing': { first: 'Tan', last: 'Rebecca, Hui Qing' },
+		'Seow Bei Yi': { first: 'Bei Yi', last: 'Seow' },
+		'Tan Dawn Wei': { first: 'Dawn Wei', last: 'Tan' },
+		'Tan Fong Han': { first: 'Fong Han', last: 'Tan' },
+		'Tan Hsueh Yun': { first: 'Hsueh Yun', last: 'Tan' },
+		'Tan Hui Yee': { first: 'Hui Yee', last: 'Tan' },
+		'Tan Jia Ning': { first: 'Jia Ning', last: 'Tan' },
+		'Tan Tam Mei': { first: 'Tam Mei', last: 'Tan' },
+		'Tan Weizhen': { first: 'Weizhen', last: 'Tan' },
+		'Tang Fan Xi': { first: 'Fan Xi', last: 'Tang' },
+		'Tay Hong Yi': { first: 'Hong Yi', last: 'Tay' },
+		'Tee Zhuo': { first: 'Zhuo', last: 'Tee' },
+		'Teo Cheng Wee': { first: 'Cheng Wee', last: 'Teo' },
+		'Tham Yuen-C': { first: 'Yuen-C', last: 'Tham' },
+		'Thong Yong Jun': { first: 'Yong Jun', last: 'Thong' },
+		'Toh Wen Li': { first: 'Wen Li', last: 'Toh' },
+		'Wong Kim Hoh': { first: 'Kim Hoh', last: 'Wong' },
+		'Zhao Jiayi': { first: 'Jiayi', last: 'Zhao' }
+	};
+	if (authorList[authorName]) {
+		newItem.creators.push({
+			lastName: authorList[authorName].last,
+			firstName: authorList[authorName].first,
+			creatorType: 'author'
+		});
+	} else {
+		newItem.creators.push(ZU.cleanAuthor(authorName, "author"));
+	}
+}
+
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "https://www.straitstimes.com/asia",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://www.straitstimes.com/advertise-with-the-straits-times",
+		"items": [
+			{
+				"itemType": "webpage",
+				"title": "Advertise with The Straits Times",
+				"creators": [],
+				"date": "2020-08-04T04:00+08:00",
+				"language": "en",
+				"url": "https://www.straitstimes.com/advertise-with-the-straits-times",
+				"websiteTitle": "The Straits Times",
+				"attachments": [
+					{
+						"title": "The Straits Times Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.straitstimes.com/singapore/fast-and-furious-can-we-trust-the-speedy-development-of-covid-19-vaccines",
+		"items": [
+			{
+				"itemType": "newspaperArticle",
+				"title": "Fast and Furious. Can we trust the speedy development of Covid-19 vaccines?",
+				"creators": [],
```

Comment:
The page lists the author as Danny Soon, so we should be scraping that.

Reply:
Ah. This is one of the guest article which doesn't have article:author meta tag.